engine.py

```diff
@@ -1041,18 +1041,48 @@ def _win32_get_proxy_window(self):
         # Get the main Photoshop window:
         ps_hwnd = self._win32_get_photoshop_main_hwnd()
         win32_proxy_win = None
+        proxy_win_hwnd = None
 
         if ps_hwnd:
             from tank.platform.qt import QtGui, QtCore
 
             # Create the proxy QWidget.
             win32_proxy_win = QtGui.QWidget()
-            win32_proxy_win.setWindowTitle('sgtk dialog owner proxy')
+            window_title = "sgtk dialog owner proxy"
```

Comment:
Is this shown anywhere? Should it be something more less technical sounding? E.g. "Shotgun Photoshop"

Reply:
It's not shown anywhere, no. However, might as well make it something more official sounding anyway!